tools/tkn-results/cmd/logs/get_log.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 The Tekton Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logs
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	pb "github.com/tektoncd/results/proto/v1alpha2/results_go_proto"
+	"github.com/tektoncd/results/tools/tkn-results/internal/flags"
+	"github.com/tektoncd/results/tools/tkn-results/internal/format"
+)
+
+func GetLogCommand(params *flags.Params) *cobra.Command {
+	opts := &flags.GetOptions{}
+
+	cmd := &cobra.Command{
+		Use: `get [flags] <log>
+
+  <log path>: Log full name to query. This is typically "<namespace>/results/<result name>/logs/<log name>".`,
+		Short: "Get Log",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			resp, err := params.LogsClient.GetLog(cmd.Context(), &pb.GetLogRequest{
+				Name: args[0],
+			})
+			if err != nil {
+				fmt.Printf("GetLog: %v\n", err)
+				return err
+			}
+			data, err := resp.Recv()
+			if err != nil {
+				fmt.Printf("Get Log Client Resp: %v\n", err)
+				return err
```

Comment:
I think there is something missing in the CLI implementation. Because REST and gRPC requests do show all the chunks. CLI is only showing the first one.

Reply:
This can be done as part of #587